### PR TITLE
model-configuration parameter and initial rework

### PR DESCRIPTION
**Pull request by blanchco**
# Description

* Rework model config parameter and intial tables
* Put in some _similar_ functions to show parameters and initials like on the model page (doesn't show a matrix for every stratified value)
* styling changes

https://github.com/DARPA-ASKEM/terarium/assets/33158416/a85aaf12-da5b-4429-a7cb-c826b9c66c38

### CODE REVIEW

File: packages/client/hmi-client/src/components/model/petrinet/tera-initial-entry.vue [hunk 1]
```diff
@@ -1,32 +1,40 @@
...
-	</header>
-	<main>
-		<span class="expression">
+	<div>
+		<header>
+			<div>
+				<strong>{{ initialId }}</strong>
+				<span v-if="name" class="ml-1">{{ '| ' + name }}</span>
+				<span v-if="unit" class="ml-2">({{ unit }})</span>
+				<template v-if="unit">
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/model/petrinet/tera-initial-entry.vue:8]: Is displaying unit information twice here intentional?

**Review by YohannParis — APPROVED** [review]

File: packages/client/hmi-client/src/components/model/petrinet/tera-initial-entry.vue (hunk 1 above)
Comment (mwdchang) [reply packages/client/hmi-client/src/components/model/petrinet/tera-initial-entry.vue:7]: Cole removed it in one of the later commit, so it probably didn't show up for you.

**Review by mwdchang — APPROVED** [review]

File: packages/client/hmi-client/src/components/model/petrinet/tera-initial-entry.vue (hunk 1 above)
Comment (blanchco) [reply packages/client/hmi-client/src/components/model/petrinet/tera-initial-entry.vue:7]: sorry forgot to comment that I removed it...

File: packages/client/hmi-client/src/components/model/petrinet/tera-initial-entry.vue [hunk 2]
```diff
@@ -1,32 +1,39 @@
 <template>
-	<header>
-		<strong>{{ initialId }}</strong>
-		<span v-if="name" class="ml-1">{{ '| ' + name }}</span>
-		<span v-if="unit" class="ml-2">({{ unit }})</span>
-		<span v-if="description" class="ml-4">{{ description }}</span>
-	</header>
-	<main>
-		<span class="expression">
+	<div>
```
[the remark this answers is not in the document]
Comment (blanchco) [reply packages/client/hmi-client/src/components/model/petrinet/tera-initial-entry.vue:2]: I wanted to pass a class into `tera-initial-entry`, it needed to be surrounded by a single element for me to do that 😬

**Pull request merged**